packages/libs/src/sdk/utils/errors.ts

```diff
@@ -5,9 +5,10 @@ export class MissingOtpUserAuthError extends Error {
   }
 }
 
-export class AntiAbuseAttestionError extends Error {
+export class AntiAbuseOracleAttestationError extends Error {
+  override name = 'AntiAbuseOracleAttestationError'
   constructor(public code: number, message: string) {
-    super(message) // Pass message to the Error class constructor
-    this.name = 'AntiAbuseAttestionError' // Set the error name to the class name
+    super(message)
+    Object.setPrototypeOf(this, AntiAbuseOracleAttestationError.prototype)
```

Comment:
What's this for?
Also do we still want to set the `name` field? I think that gets used in generating exception output for things like sentry.

Reply:
Don't I do that above in the override?

Reply:
I did not see that line, was focused on the removal of `this.name = ...`
Carry on! 🤦